Gluing/Conservativity.agda

```diff
@@ -145,19 +145,22 @@ module _ (Q : Quiver ℓQ ℓQ') where
       λ f → funExt (λ _ → ΣPathP (FREE .⋆Assoc _ _ _ ,
         isSet→SquareP (λ _ _ → FREE-1,× .fst .isSetHom) _ _ _ _))
 
-  ⊆-Full : isFull ⊆
-  ⊆-Full o o' F[f] = ∣ f , p ∙ FREE-1,× .fst .⋆IdL _ ∣₁
-    where
-    ⊆[→o'] : 𝓟FREEᴰ.ob[ nerve ⟅ ⊆ ⟅ o' ⟆ ⟆ ]
-    ⊆[→o'] = S .F-obᴰ (⊆ ⟅ o' ⟆)
-    ⊆[→o']* : 𝓟FREEᴰ.ob[ nerve ⟅ ⊆ ⟅ o ⟆ ⟆ ]
-    ⊆[→o']* = PRESHEAFᴰ-AllCartesianOvers _ _ _ ⊆[→o'] (nerve ⟪ F[f] ⟫) .f*cᴰ'
-    f,p : ⟨ ⊆[→o']* ⟅ o , FREE-1,× .fst .id ⟆ ⟩
-    f,p = (S .F-homᴰ F[f] ⟦ o , FREE-1,× .fst .id ⟧) (FREE .id , refl)
-    f : FREE [ o , o' ]
-    f = f,p .fst
-    p : ⊆ ⟪ f ⟫ ≡ FREE-1,× .fst .id ⋆⟨ FREE-1,× .fst ⟩ F[f]
-    p = f,p .snd
+  opaque
```

Comment:
(I just started using this last week and this was just a mental note that this "leaf" theorem is opaque, because well, the type is the theorem, the proof shouldn't be expanded and used elsewhere)

Reply:
(I used opaque to try to fix a performance problem although I'm not clear if it helped or not. The proof got done and includes opaque lemmas, but maybe didn't contribute to the proof getting done.)